.github/workflows/reusable-release.yml

```diff
@@ -315,10 +329,9 @@ jobs:
         env:
           RELEASE_TAG: ${{ needs.push-release-commit.outputs.release_tag }}
           RELEASE_COMMIT_ID: ${{ needs.push-release-commit.outputs.release_commit_id }}
-          ARTIFACT_SHA256SUMS: ${{ needs.create-artifacts.outputs.ARTIFACT_SHA256SUMS }}
           KEY_EMAIL: ${{ needs.init.outputs.key_email }}
         run: |
-          cd repo
+          cd $GITHUB_WORKSPACE/repo
```

Comment:
Here we need to use an absolute path, rather than a relative one, because the new hash verification stage has already shifted directory from the original [`GITHUB_WORKSPACE`](https://docs.github.com/en/actions/learn-github-actions/variables#:~:text=the%20workflow%20file.-,GITHUB_WORKSPACE,-The%20default%20working) folder